src/main/scala/com/rbmhtechnology/calliope/SequencedMessage.scala

```diff
@@ -16,4 +16,11 @@
 
 package com.rbmhtechnology.calliope
 
-case class SequencedMessage[A](payload: A, sourceId: String, sequenceNo: Long, creationTimestamp: Long)
\ No newline at end of file
+case class SequencedMessage[A](payload: A, sourceId: String, sequenceNo: Long, creationTimestamp: Long)
```

Comment:
We are in the context of an **event** bus, so I think it's a good name. We will anyway need a bigger redesign round later where we can make a distinction between general messages and events. But for now, let's narrow the scope to events.